src/lib/redux/siteData.ts

```diff
@@ -34,4 +35,8 @@ export const updateTariffPlanInRedux = (tariffPlan: Partial<TariffPlanRecord>) =
 
 export const updateSiteInRedux = (siteUpdates : Partial<SiteRecord>) => {
   store.dispatch(updateSite(siteUpdates))
-}
\ No newline at end of file
+}
+
+export const selectCurrentSiteProfit = () => {
```

Comment:
Make sure that this calculates the totalProfit for the period and not total profit over all the periods!